proto/group.proto

```diff
@@ -1,17 +1,36 @@
-message GoogleGroup {
-    required string group_id = 1;
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package proto.api;
+
+option go_package = "github.com/abcxyz/team-link/apis/v1alpha3/proto;api";
+
+message GitHub {
+    int64 org_id = 1;
+    int64 team_id = 2;
 }
 
-message GitHubTeam {
-    required int64 org_id = 1;
-    required int64 team_id = 2;
+message GitLab {
+    int64 group_id = 1;
 }
 
-message GoogleGroupToGitHubTeamMapping {
-    required GoogleGroup google_group = 1;
-    required GitHubTeam git_hub_team = 2;
+message GoogleGroups {
+    string group_id = 1;
 }
 
-message GoogleGroupToGitHubTeamMappings {
-    repeated GoogleGroupToGitHubTeamMapping mappings = 1;
+message Ganpati {
```

Comment:
We probably should remove references to Ganpati and only have this in the internal version.

Reply:
removed